packages/core/src/RenderingEngine/VolumeViewport.ts

```diff
@@ -344,7 +344,27 @@ class VolumeViewport extends BaseVolumeViewport {
    * @returns The slice index
    */
   public getCurrentImageIdIndex = (): number | undefined => {
-    return this._getImageIdIndex();
+    if (!this.getCurrentImageId()) {
```

Comment:
this method should not care if it is acquisition or not, this probably will break imagIndex on the non acquisition viewports.

Reply:
Ok will change.